xdsl/printer.py

```diff
@@ -356,7 +356,15 @@ def print_complex_float(
     def print_complex_int(
         self, value: tuple[int, int], type: ComplexType[ComplexElementCovT]
```

Comment:
I forget, why is this not 
```suggestion
        self, value: tuple[int, int], type: ComplexType[IntegerType]
```
?